src/pytorch_lightning/_graveyard/__init__.py

```diff
@@ -17,4 +17,5 @@
 import pytorch_lightning._graveyard.legacy_import_unpickler
 import pytorch_lightning._graveyard.loggers
 import pytorch_lightning._graveyard.trainer
-import pytorch_lightning._graveyard.training_type  # noqa: F401
+import pytorch_lightning._graveyard.training_type
+import pytorch_lightning._graveyard.utilities
```

Comment:
pre-commit has several failures, one of them being that you need to keep the noqa in the last line